### ISSUE TITLE
Cannot parse unity ProjectSettings.asset

### ISSUE THREAD

**Issue by jamie-healey-xr** [cake-contrib/Cake.Yaml#50]
## Expected Behaviour
Attempting to parse `ProjectSettings.asset` unity file

## Current Behavior
I get Error: (Line: 3, Col: 5, Idx: 45) - (Line: 4, Col: 1, Idx: 55): Encountered an unresolved tag 'tag:unity3d.com,2011:129'.

I do understand unity only uses a subset of the yaml spec, but i'd still expect it to work.

The start of the file (containing the tag it's failing on)

```
%YAML 1.1
%TAG !u! tag:unity3d.com,2011:
--- !u!129 &1
PlayerSettings:
  m_ObjectHideFlags: 0
  serializedVersion: 23
```
I can provide a full file if needs be

## Your Environment

* Addin version used:  4.0.0
* Cake Version used:  Frosting 1.3.0
* Operating System: Windows 10.0.18362 Build 18362

**Comment by augustoproiete** [comment]
Hey @jamie-healey-xr `Cake.Yaml` leverages [`YamlDotNet`](https://github.com/aaubry/YamlDotNet) under the hood for parsing YAML content, so we're limited to what YamlDotNet supports...

It seems your file is not valid YAML, so one alternative could be to delete the first 3 lines of the file yourself, before trying to parse it.

You can also try to [open an issue in their repo](https://github.com/aaubry/YamlDotNet/issues) with a reproducible example, and try to get YamlDotNet to support these kind of files in a future version of YamlDotNet...

**Comment by jamie-healey-xr** [comment]
Thanks. I found this post that explains it in a bit more detail, for anybody else that finds this:

https://stackoverflow.com/questions/21473076/pyyaml-and-unusual-tags

TLDR: Some of Unity's YAML isn't compliant to the spec. Some hacks can be done to get around it though
